### ISSUE TITLE
Contrast ratio for focused button in TeachingBubble fails contrast ratio

### ISSUE THREAD

**Issue by jamesdeancalhoun** [microsoft/fluentui#18562]
### Environment Information

- **Package version(s)**: 8.18.0
- **Browser and OS versions**: Edge 91.0.864.41, Windows 10 Enterprise 19043.1052

### Describe the issue:

The contrast ratio (when keyboard focused) doesn't meet required minimum contrast ratio. It's 4.46 contrast ratio between the text and background of the button.

#e7efff background color
#2266e3 text color 
![image](https://user-images.githubusercontent.com/17098577/121970312-1edcb280-cd2b-11eb-879b-2599977ba948.png)

### Please provide a reproduction of the issue in a codepen:

Default codepen reproduces here: https://developer.microsoft.com/en-us/fluentui#/controls/web/teachingbubble

#### Actual behavior:

Contrast ratio is below minimum at 4.46

#### Expected behavior:

The contrast ratio between the text and the background should be 4.5:1 for font size less than 18 points.

### Documentation describing expected behavior

Described by the MAS 1.4.3 guideline

**Comment by miroslavstastny** [comment]
@smhigley could you please check?

**Comment by gouttierre** [comment]
@jamesdeancalhoun - thanks for filing this issue with us.

@smhigley - Here is another accessibility issue with color contract. 
cc @ling1726

**Comment by smhigley** [comment]
@jamesdeancalhoun could you provide a codepen that repros those colors? On our docs site page (https://developer.microsoft.com/en-us/fluentui#/controls/web/teachingbubble), the TeachingBubble examples have the following color values for foreground/background of the primary button when focused:

- background: rgb(222, 236, 249), which translates to #deecf9
- text: rgb(0, 90, 158), which translates to #005a9e

The contrast ratio for that text:bg is 5.91, which easily passes the 4.5 minimum.

**Comment by smhigley** [comment]
Here's a screenshot of the focused button ("try it out") with Edge devtools showing the rgb values:

![screenshot of the TeachingBubble open, with devtools showing the focused primary button colors](https://user-images.githubusercontent.com/3819570/128107024-c4bac47d-23f6-4a54-a370-f9d45e2450ce.png)
